### PR TITLE
Update quickstart-upgrade-win10.md

### LINKED ISSUE #1490: Broken Link (quickstart-upgrade-win10.md)
The file:
SCCMdocs\sccm\comanage\quickstart-upgrade-win10.md

Has the following broken link:
https://www.microsoft.com/download/details.aspx?id=50377

### PR DESCRIPTION
**Pull request by AndrewJNet**
Updated link to resolve #1490

#MMS2019Docathon

### Summarize the change in the pull request title

Updated link to resolve #1490

Fixes #1490 (if necessary)

### CODE REVIEW

**Comment by PRMerger12** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by AndrewJNet** [comment]
#MMS2019Docathon

**Comment by aczechowski** [comment]
Thanks @AndrewJNet for the contribution! I'll review after the end of the contest.

#hold-off
(reviewers, please don't edit - I'll take care of it.)

**Comment by AndrewJNet** [comment]
@aczechowski Should I mark #MMS2019Docathon on the issue itself or is it sufficient in the PR?

**Comment by aczechowski** [comment]
Sufficient on the PR, thanks for asking 😄

File: sccm/comanage/quickstart-upgrade-win10.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Here are some tips we've learned from helping thousands of customers get current
 
 ## Case studies
 
-Microsoft IT deployed Windows 10 to 96,000 distributed users at Microsoft. The deployment included both remote users and users on the corporate network. The deployment completed in nine weeks. For more information on their experience, see [Deploying Windows 10 at Microsoft as an in-place upgrade](https://www.microsoft.com/download/details.aspx?id=50377).  
+Microsoft IT deployed Windows 10 to 96,000 distributed users at Microsoft. The deployment included both remote users and users on the corporate network. The deployment completed in nine weeks. For more information on their experience, see [Deploying Windows 10 at Microsoft as an in-place upgrade](https://www.microsoft.com/en-us/itshowcase/deploying-windows-10-at-microsoft-as-an-in-place-upgrade).
```
Comment (aczechowski) [inline sccm/comanage/quickstart-upgrade-win10.md]: Remove the locale (`en-us/`) from the URL so that it works for any language

**Comment by aczechowski** [comment]
Ping @AndrewJNet can you review and revise?

**Comment by PRMerger7** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @AndrewJNet 
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 22 May) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**